src/pyskindose/phantom_class.py

```diff
@@ -2,13 +2,26 @@
 import numpy as np
```

Comment:
Separate standard library and third party imports

Reply:
resolved